### ISSUE TITLE
`unstable.check-cfg` expected true/false, but found a array

### ISSUE THREAD

**Issue by yopox** [boozook/playdate#221]
Hello, thanks for all the work on playdate x rust!
I tried to setup a new project but it was unfortunately not so easy:

```bash
╭─yopox ~/PlaydateProjects
╰─➤  mkdir playdate-shooter

╭─yopox ~/PlaydateProjects
╰─➤  cd playdate-shooter

╭─yopox ~/PlaydateProjects/playdate-shooter
╰─➤  cargo playdate init --lib --full-metadata --deps="playdate"
    Creating library (requested).
    Creating library package
note: see more `Cargo.toml` keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
    Finished finally 🎉

╭─yopox ~/PlaydateProjects/playdate-shooter  ‹master*›
╰─➤  rustup -V
rustup 1.27.0 (bbb9276d2 2024-03-08)
info: This is the version for the rustup toolchain manager, not the rustc compiler.
info: The currently active `rustc` version is `rustc 1.78.0-nightly (f4b771bf1 2024-03-14)`

╭─yopox ~/PlaydateProjects/playdate-shooter  ‹master*› 
╰─➤  cargo -V                                                                                                                                                 1 ↵
cargo 1.78.0-nightly (7065f0ef4 2024-03-12)

╭─yopox ~/PlaydateProjects/playdate-shooter  ‹master*›
╰─➤  cargo build
error: error in /Users/yopox/PlaydateProjects/playdate-shooter/.cargo/config.toml: `unstable.check-cfg` expected true/false, but found a array

╭─yopox ~/PlaydateProjects/playdate-shooter  ‹master*›
╰─➤  cargo playdate run                                                                                                                                    101 ↵
error: error in /Users/yopox/PlaydateProjects/playdate-shooter/.cargo/config.toml: `unstable.check-cfg` expected true/false, but found a array
```

It looks like there is an issue with my configuration or with the sample project configuration.

**Comment by boozook** [comment]
Oops 🤦🏻‍♂️, that's because couple versions back breaking changes in cargo happens. So, imagine, you have your cargo in the toolchain and `cargo-playdate` that build on cargo, but on other version.
I'll fix it soon.

Possible workaround:
Try to remove `unstable.check-cfg` from config.
Actually you can remove config at all without regret or loss. So try `rm ~/PlaydateProjects/playdate-shooter/.cargo/config.toml`.

**Comment by boozook** [comment]
@yopox, plz try fixed [version 0.3.16](https://crates.io/crates/cargo-playdate/0.3.16).
